src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example6.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="com.puppycrawl.tools.checkstyle.filters.suppressionxpathsinglefilter.MyMethod">
+      <message key="name.invalidPattern" value="Name ''{0}'' must match pattern ''{1}''."/>
```

Comment:
lets avoid custom modules.
Goal of example is `To suppress checks in the FileOne file by non-query: `
so we can use any existing Check you like.